tox.ini

```diff
@@ -43,7 +43,74 @@ deps =
+[testenv:marathon_itests]
+basepython = python2.7
+changedir=paasta_itests/
+passenv = DOCKER_TLS_VERIFY DOCKER_HOST DOCKER_CERT_PATH
+deps =
+    docker-compose==1.3.0
+commands =
+    docker-compose pull --allow-insecure-ssl
+    docker-compose --verbose build
+    # Fire up the marathon cluster in background
+    docker-compose up -d mesosmaster mesosslave marathon chronos hacheck
+    # Run the paastatools container in foreground to catch the output
+    # the `docker-compose run` vs `docker-compose up` is important here, as docker-compose run will
+    # exit with the right code.
+    #
+    # dnephin says we need the --rm otherwise these containers won't be cleaned
+    # up. I guess we only need this for run'd containers, not up'd containers?
+    # IDK, the docs don't really specify.
+    docker-compose run --rm paastatools tox -e paasta_itests_inside_container -- --no-capture --tags marathon {posargs}
+    docker-compose stop
+    docker-compose rm --force
+
+[testenv:chronos_itests]
+basepython = python2.7
+changedir=paasta_itests/
+passenv = DOCKER_TLS_VERIFY DOCKER_HOST DOCKER_CERT_PATH
+deps =
+    docker-compose==1.3.0
+commands =
+    docker-compose pull --allow-insecure-ssl
+    docker-compose --verbose build
+    # Fire up the marathon cluster in background
+    docker-compose up -d mesosmaster mesosslave marathon chronos hacheck
+    # Run the paastatools container in foreground to catch the output
+    # the `docker-compose run` vs `docker-compose up` is important here, as docker-compose run will
+    # exit with the right code.
+    #
+    # dnephin says we need the --rm otherwise these containers won't be cleaned
+    # up. I guess we only need this for run'd containers, not up'd containers?
+    # IDK, the docs don't really specify.
+    docker-compose run --rm paastatools tox -e paasta_itests_inside_container -- --no-capture --tags chronos {posargs}
+    docker-compose stop
+    docker-compose rm --force
+
+
+[testenv:everything_else_itests]
+basepython = python2.7
+changedir=paasta_itests/
+passenv = DOCKER_TLS_VERIFY DOCKER_HOST DOCKER_CERT_PATH
+deps =
+    docker-compose==1.3.0
+commands =
+    docker-compose pull --allow-insecure-ssl
+    docker-compose --verbose build
+    # Fire up the marathon cluster in background
+    docker-compose up -d mesosmaster mesosslave marathon chronos hacheck
+    # Run the paastatools container in foreground to catch the output
+    # the `docker-compose run` vs `docker-compose up` is important here, as docker-compose run will
+    # exit with the right code.
+    #
+    # dnephin says we need the --rm otherwise these containers won't be cleaned
+    # up. I guess we only need this for run'd containers, not up'd containers?
+    # IDK, the docs don't really specify.
+    docker-compose run --rm paastatools tox -e paasta_itests_inside_container -- --no-capture --tags ~marathon,~chronos {posargs}
+    docker-compose stop
+    docker-compose rm --force
+
+[testenv:all_itests]
```

Comment:
Instead of duplicating this code 3 times, you could just pass `--tags chronos`, etc. to `tox -e paasta_itests` in travis.yml.